script.js

```diff
@@ -1,4 +1,285 @@
+          currentTotal += 11;
+          aceCounter += 1;
+        } else {
+          currentTotal += allCardsDealt[playerTurn - 1][i].blackjackValue;
+        }
+      }
+      if (currentTotal == 21) {
+        myOutputValue += dealerCards;
+        myOutputValue += `<br><br>Dealer has scored a <b>Blackjack</b> ✨! Press '<b>Submit</b>' to see results...`;
+        playersBlackjackValue[playerTurn - 1] = "Blackjack";
+      } else {
+        while (currentTotal <= 16) {
+          dealerCards = "";
+          allCardsDealt[playerTurn - 1].push(newDeck.pop());
+          var currentTotal = 0;
+          var aceCounter = 0;
+          for (var i = 0; i < allCardsDealt[playerTurn - 1].length; i += 1) {
+            dealerCards += `<br>${allCardsDealt[playerTurn - 1][i].name} of ${
+              allCardsDealt[playerTurn - 1][i].emoji
+            }`;
+            if (
+              allCardsDealt[playerTurn - 1][i].name == "ace" &&
+              aceCounter == 0
+            ) {
+              aceCounter += 1;
+              currentTotal += 11;
+            } else {
+              currentTotal += allCardsDealt[playerTurn - 1][i].blackjackValue;
+            }
+          }
+          if (aceCounter != 0 && currentTotal > 21) {
+            currentTotal -= 10;
+          }
+        }
+        myOutputValue += dealerCards;
+        myOutputValue += `<br><br>Total dealer card value is <b>${currentTotal}</b>`;
+        if (currentTotal > 21) {
+          myOutputValue += ` (That looks like a <b>bust</b>! 💣)`;
+        }
+        myOutputValue += `<br>Press '<b>Submit</b>' to see results...`;
+        if (currentTotal > 21) {
+          playersBlackjackValue[playerTurn - 1] = 0;
+        } else {
+          playersBlackjackValue[playerTurn - 1] = currentTotal;
+        }
+      }
+      gameMode = "results";
+    } else {
+      // Regular Player's turn (NOT DEALER)
+      myOutputValue += `<b><u>Player ${playerTurn}</b></u><br>`;
+      myOutputValue += `Your cards are:`;
+      var currentTotal = 0;
+      var aceCounter = 0;
+      for (var i = 0; i < allCardsDealt[playerTurn - 1].length; i += 1) {
+        myOutputValue += `<br>${allCardsDealt[playerTurn - 1][i].name} of ${
+          allCardsDealt[playerTurn - 1][i].emoji
+        }`;
+        if (allCardsDealt[playerTurn - 1][i].name == "ace" && aceCounter == 0) {
+          currentTotal += 11;
+          aceCounter += 1;
+        } else {
+          currentTotal += allCardsDealt[playerTurn - 1][i].blackjackValue;
+        }
+      }
+      if (currentTotal == 21) {
+        myOutputValue += `<br><br>WOW! You got a <b>Blackjack</b> ✨! Press '<b>Submit</b>' to continue to next player...`;
+        playerTurn += 1;
+        playersBlackjackValue.push("Blackjack");
+      } else {
+        myOutputValue += `<br><br>Your total card value is <b>${currentTotal}</b>`;
+        myOutputValue += `<br>Do you want to '<b>hit</b>' 💥 or '<b>stand</b>' ✋🏼?<br><i>(p.s. You must have at least 17 to stand)</i>`;
```

Comment:
could be assigned once, but you could have separated it into 2 lines for increase readability